libs/input-mapping/tests/Functional/DownloadTablesRedshiftTest.php

```diff
@@ -20,6 +20,7 @@ class DownloadTablesRedshiftTest extends AbstractTestCase
     #[NeedsTestRedshiftTable]
     public function testReadTablesRedshift(): void
     {
+        self::markTestSkipped('does not work');
```

Comment:
spíš bych řekl, že blbne redshift, ale nemíním to teď zkoumat
https://keboola.atlassian.net/browse/SOX-228